### PR TITLE
Redux optimisation

### LINKED ISSUE #93: Optimising the Feed
The Feed can take quite a while to render, and in some cases it can block the browser while rendering.

### PR DESCRIPTION
**Pull request by Winwardo**
Closes #93

### CODE REVIEW

**Comment by Winwardo** [comment]
@VasilyShelkov ready for comments and merge

File: src/client/results/Feed.js [hunk 1]
```diff
@@ -46,32 +46,30 @@ Feed.propTypes = {
...
-        </div>
+let PaginationButtons = ({ dispatch, numberOfPages, paginationInfo }) => (
+  <div className="ui grid">
+    <div className="two column row">
+      <div className="left column">
+        <LimitButtons paginationInfo={paginationInfo} dispatch={dispatch} /> results per page.
+      </div>
+      <br />
+      <div className="right aligned column">
+        <PagePicker numberOfPages={numberOfPages} paginationInfo={paginationInfo} dispatch={dispatch} />
```
Comment (VasilyShelkov) [inline src/client/results/Feed.js]: I know I should have pickedd this up last PR, but can you change passing down dispatch to just passing down 
`(limit) => {dispatch(setFeedPageLimit(limit));}`

File: src/client/results/Results.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ import { connect } from 'react-redux';
 import Feed from './Feed';
 import MostUsedWords from './mostfrequent/words/MostUsedWords';
 import MostActiveUsers from './mostfrequent/users/MostActiveUsers';
-import { groupedCountWords, mostFrequentWords, mostFrequentUsers } from './../tweetAnalysis';
+import { mostFrequentUsers } from './../tweetAnalysis';
```
Comment (VasilyShelkov) [inline src/client/results/Results.js]: can you remove unused mostFrequentUsers import

File: src/client/results/mostfrequent/words/MostUsedWords.js [hunk 3]
```diff
@@ -16,7 +16,7 @@ const MostUsedWords = ({ dispatch, wordInfoList, filterTerm }) => {
       }}
       filterTerm={filterTerm}
     >
-      <WordItemsList words={filteredItems} />
+      <WordItemsList words={filteredItems} dispatch={dispatch} />
```
Comment (VasilyShelkov) [inline src/client/results/mostfrequent/words/MostUsedWords.js]: Same comment as above, don't pass down dispatch but the specific function

**Pull request merged**